YvesIraguha/lost-and-found-api
# Enable docker app container to run

#### What does this PR do?
Fix Docker services
#### Description of Task to be completed?
- Fix available services
- Add mongo GUI services for data visualization
#### How should this be manually tested?
- Clone the repo `git clone https://github.com/YvesIraguha/lost-and-found-api.git`
- Switch your working directory `cd lost-and-found-api`
- Checkout this branch `git checkout bg-fix-docker`
- Build the image `docker-compose build`
- Start the container `docker-compose up`
- In your browser hit on `http://localhost:3000/api/v1/users/google`
> This should prompt on google login on the app
> After successful login you should get your data and the token
- Still in the browser, hit on `http://localhost:8081` to use mongoGUI to see your data
#### Any background context you want to provide?
- Make sure docker desktop is installed on your PC
